Article/Article.js

```diff
@@ -2,19 +2,25 @@
 
 class Article {
   constructor(domElement) {
+
+    
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
+   
+   
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'Expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
```

Comment:
You could also have done es6 syntax here 
` this.expandButton.addEventListener('click', () => { this.expandArticle() });`